spotinst/resource_spotinst_ocean_right_sizing_rule.go

```diff
@@ -380,15 +401,82 @@ func expandWorkloads(data interface{}) ([]*right_sizing.Workload, error) {
 			workload.RegexName = spotinst.String(v)
 		}
 
-		workloads = append(workloads, workload)
+		//During first workload API call responseDataWorkload, responseDataRegex and responseDataLabel will be empty list, hence not validating user's workload again GET attachedWorkload response in IF block.
+		//In case of responseDataWorkload, responseDataRegex empty list not going to validate user's workload again GET attachedWorkload response in IF block.
```

Comment:
correct the spelling of against and second statement is not clear

Reply:
Updating comment to :
//Since responseDataWorkload, responseDataRegex and responseDataLabel will be empty list in GET attachedWorkload API response, so validation of user's workload against GET attachedWorkload response in IF block is not required in case of first attach workload API call.